spec/yaml/swiftnav/sbp/file_io.yaml

```diff
@@ -75,9 +95,28 @@ definitions:
           size: 20
           desc: Name of the directory to list (NULL padded)
 
+ - MSG_FILEIO_READ_DIR_RESPONSE:
+    id: 0x00AA
+    short_desc: Files listed in a directory (host <= device)
+    desc: |
+      The read directory message lists the files in a directory on the
+      device's onboard flash file system.  The offset parameter can be
+      used to skip the first n elements of the file list. Message contains
+      the directory listings as a NULL delimited list. The listing is
+      chunked over multiple SBP packets and the end of the list is
+      identified by an entry containing just the character 0xFF.
+    fields:
+      - offset:
+          type: u32
+          desc: |
+            The offset to skip the first n elements of the file list
+      - dirname:
+          type: string
+          size: 20
+          desc: Name of the directory to list (NULL padded)
+
```

Comment:
Same here - where is the field for the actual data (i.e the result of the "dir" command)  Are people supposed to parse that out of the payload themselves?